data/kahet/aberrant outfits.txt

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2023 by Azure
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Weapons:
+
+
+outfit "Aberrant Spiker"
+	category "Secondary Weapons"
+	licenses
+		"Twilight Guard"
+	cost 1370000
+	thumbnail "outfit/unknown"
+	"mass" 6
+	"outfit space" -15
+	"weapon capacity" -15
+	"gun ports" -1
+	"nettle capacity" 60
+	weapon
+		sprite "projectile/aberrant spike"
+			"frame rate" 24
+			"scale" 0.5
+			"rewind"
+		sound "finisher"
+		icon "icon/finisher"
+		"fire effect" "sidewinder fire"
+		"fire effect" "speckle fire"
+		"inaccuracy" 2.4
+		"reload" 60
+		"firing energy" 6
+		"firing heat" 3
+		"velocity" 6
+		"lifetime" 1800
+		"acceleration" 0.6
+		"drag" .09
+		"turn" 3
+		"homing" 3
+		"optical tracking" 0.8
+		"hit force" 300
+		"hull damage" 1000
+		"shield damage" 1000
+		"missile strength" 12
+	description "Spikes grow on the backs of many Aberrant variants, made of a seemingly random assortment of materials both metallic and mineral in nature. This device appears to allow an Aberrant to launch these spikes as an offensive weapon, but just how they are able to guide and propel themselves through space is a mystery that a cursory examination of this outfit does not reveal."
+
+
+outfit "Nucleolysis Beam"
```

Comment:
Nearly 1000 effective hull DPS from a 23 ton weapon is too much, especially for a weapon that also does a considerable amount of fuel damage. The low shield damage is somewhat of a downside in many cases, but I think this weapon with these stats would be far too strong if used with disruption.